### DISCUSSION TITLE
Handling Large Numbers and Precision

### DISCUSSION THREAD

**Discussion by adesokan01** [adesokan01/T24S2T1EXTG01#6] [General]
How should we handle large currency amounts and precision in calculations? Should there be a limit on the maximum amount the user can enter, and how many decimal places should we consider for display and internal calculations to ensure accuracy without floating-point errors?

**Comment by acantley** [comment]
I don't think there needs to be a limit on the maximum amount a user can enter. Internally, from my understanding, using the **decimal** type, rather than the **float** type is going to help ensure the accuracy of internal calculations. As for display, given the currencies we are dealing with, I think it most appropriate to display the result to the user with two decimal places.
